.github/workflows/ci.yml

```diff
@@ -0,0 +1,44 @@
+
+on:
+  push:
+    branches:
+     # - development
+      - testpipeline
+  pull_request:
+    branches:
+      - testpipeline
+
+name: Sslscan Workflow
+
+jobs:
+  check-format:
+    name: Check format
+    runs-on: ubuntu-latest
+    container: crystallang/crystal
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Check sources format
+        run: crystal tool format --check
+
+  unit-tests:
+    name: Unit tests
+    runs-on: ubuntu-latest
+    container: crystallang/crystal
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1      
+      - name: Unit testing
+        run: shards install && crystal spec && crystal bin/ameba.cr
```

Comment:
It's OK to divide this line on three steps:

1. Shards installing
2. Crystal spec running
3. Ameba

In that case, you can remove ameba job